core/src/test/java/org/apache/calcite/jdbc/CalciteRemoteDriverTest.java

```diff
@@ -129,6 +129,17 @@ class CalciteRemoteDriverTest {
         }
       };
 
+  private static final Function<Connection, ResultSet> GET_TABLES =
```

Comment:
I agree it's strange for Java. Modern java uses method references, e.g. `CalciteRemoteDriverTest::getTables`, rather than constant lambdas. But it's consistent with the rest of the code in this class.